### PR TITLE
GitBash_Code_So_Far_#10

### PR DESCRIPTION
**Pull request by mauricio-ocampo**
This is a Review of the Code and explanation of the command lines used in GitBash so far. From the configuration and setup up to date. Hope it helps to get used to the new Workflow.

### CODE REVIEW

File: Codes_GitBash_So_Far.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+#Uploading all local commits (changes) to the GitHub server
+git push
+
+#Creating a new File in the actual Folder and open it with the editor
+start notepad++ Codes_GitBash_So_Far.md
+
+#Removing a File 
+rm Codes_GitBash_So_Far.md
+#Or
+git rm -a Codes_GitBash_So_Far.md
```
Comment (Braak-1991) [inline Codes_GitBash_So_Far.md:65]: When removing a file you also need to commit the removal just as with adding.

File: Codes_GitBash_So_Far.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
+Steps done until now in GitBash and GitHub
```
Comment (Braak-1991) [inline Codes_GitBash_So_Far.md:1]: If you use a markdown file, you need to add a space after # if you want to display a headline

Comment (Braak-1991) [reply Codes_GitBash_So_Far.md:1]: see [Cheatsheet for markdown](https://github.com/htw-pv3/literature/blob/master/Cheatsheet_MD.mdl)

File: Codes_GitBash_So_Far.md (hunk 1 above)
Comment (mauricio-ocampo) [reply Codes_GitBash_So_Far.md:65]: - you mean I need to add the command lines:?
git add git_cheat_sheet.md
git commit -m "Deleting the File #6"
-Or in one line
git commit -a -m "Deleting the File #6"

Comment (Braak-1991) [reply Codes_GitBash_So_Far.md:65]: like this:
`git rm file1.txt`
`git commit -m "remove file1.txt"`

Comment (mauricio-ocampo) [reply Codes_GitBash_So_Far.md:65]: - you mean adding the Following command lines:?
git add git_cheat_sheet.md
git commit -m "Deleting the File #6"
- Or in one line both commands
git commit -a -m "Deleting the File #6"

**Pull request merged**